pkg/controller/appsync/appsync.go

```diff
@@ -45,7 +46,23 @@ func New(config *config.AppSyncConfig, db *database.Database, h *render.Renderer
 // HandleSync performs the logic to sync mobile apps.
 func (c *Controller) HandleSync(ctx context.Context) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		// TODO(whaught): implement this
-		controller.InternalError(w, r, c.h, errors.New("not implemented"))
+		resp, err := http.Get(c.config.AppSyncURL)
+		if err != nil {
+			controller.InternalError(w, r, c.h, err)
+			return
+		}
+
+		defer resp.Body.Close()
+		body, err := ioutil.ReadAll(resp.Body)
```

Comment:
We need to add more diligence and resilience here. Check how we do some of the downloading elsewhere. We should put a max size reader on this too.